workshops/collab_sketch/README.md

```diff
@@ -82,6 +82,41 @@ firebase.initializeApp(config);
 var pointsData = firebase.database().ref();
 ```
 
+Let's quickly walk through what a part of this code does.
+
+```js
+var config = {
+  (...) // stuff hidden here for simplicity
+}
+```
+
+This section of code creates a new `object` in JavaScript named `config`.  You can think of `object`s like containers for things like text or numbers which we call `properties`. 
+
+`properties` are values that we store by name in the `object` and then retrieve at a later time.
+
+```js
+var config = {
+  apiKey: "AIxaSyGsAkHke9lXEU_97a8rYpMn7gOH3eWDxrM",
+  (...)
+}
+```
+
+The first thing we've added to the object is a `property` named `apiKey`, which we assigned a jumbled value of text to.  To access this value, we can write:
+
+```js
+config.apiKey
+```
+
+The `.` is a special character that allows us to look inside of an `object`.  This can be used as a shortcut to replacing code, so writing `config.apiKey` is many times equivalent to writing the full jumbled text `"AIxaSyGsAkHke9lXEU_97a8rYpMn7gOH3eWDxrM"`.
+
+In the same manner, `config.authDomain` would look inside to the `object` for the value of `authDomain` and become "replaced" by `"collab-draw.firebaseapp.com"`.
```

Comment:
"become replaced by" -> "receive/take on the value of" or something? I was a little confused by "replaced."